File: baekjoon1010.py
T = int(input())
for _ in range(T):
    N, M = [int(x) for x in input().split()]
    j = 1
    rst = 1
    for i in range(M-N+1,M+1):
        rst *= i
        rst /= j
        j += 1
    print(int(rst))

File: baekjoon1010.c
#include<stdio.h>
int main(){
    int t,n,m;
    int rst,k;
    scanf("%d", &t);
    for(int i=0;i<t;i++){
        rst = 1;
        k = 1;
        scanf("%d %d", &n, &m);
        for(int j=m;j>m-n;j--){
            rst *= j;
            rst /= k++;
        }
        printf("%d\n", rst);
    }
    return 0;
}
